lib/session-store.js

```diff
@@ -0,0 +1,74 @@
+/*
+// https://github.com/expressjs/session#session-store-implementation
+//
+// Uses PL database as a express-session store
+*/
+const session = require('express-session');
+const _ = require('lodash');
+const util = require('util');
+
+const config = require('./config');
+const sqldb = require('@prairielearn/postgres');
+const sql = sqldb.loadSqlEquiv(__filename);
+
+class plSessionStore extends session.Store {
```

Comment:
```suggestion
class SessionStore extends session.Store {
```

Alternatively, `PLSessionStore` (class names should be capitalized), but I don't think we really need the "PL" branding on things.